docs/config.rst

```diff
@@ -110,7 +110,20 @@ Core configuration
     The original MPD server only supports 10000 tracks in the tracklist. Some
     MPD clients will crash if this limit is exceeded.
 
-
+.. confval:: core/restore_state
```

Comment:
Since we are getting closer to merging, anyone have any more thoughts around this?